notebooks/test/executed_notebooks/BigQuery Basics_output.ipynb

```diff
@@ -0,0 +1,615 @@
+    "    skip_leading_rows=1,\n",
+    "    autodetect=True\n",
+    ")\n",
+    "\n",
+    "with open(source_filename, 'rb') as source_file:\n",
+    "    job = client.load_table_from_file(\n",
+    "        source_file,\n",
+    "        table_ref,\n",
+    "        location='US',  # Must match the destination dataset location.\n",
+    "        job_config=job_config)  # API request\n",
+    "\n",
+    "job.result()  # Waits for table load to complete.\n",
+    "\n",
+    "print('Loaded {} rows into {}:{}.'.format(\n",
+    "    job.output_rows, dataset_id, table_ref.path))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load data from Google Cloud Storage to a table\n",
+    "\n",
+    "The example below demonstrates how to load a local CSV file into a new or existing table. See [SourceFormat](https://googleapis.github.io/google-cloud-python/latest/bigquery/generated/google.cloud.bigquery.job.SourceFormat.html#google.cloud.bigquery.job.SourceFormat) in the Python client library documentation for a list of available source formats. For more information, see [Introduction to Loading Data from Cloud Storage](https://cloud.google.com/bigquery/docs/loading-data-cloud-storage) in the BigQuery documentation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Starting job f65c1470-3e8b-4b0f-8a44-ff6a3f5cbaf0\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Job finished.\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Loaded 50 rows.\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Configure the load job\n",
+    "job_config = bigquery.LoadJobConfig(\n",
+    "    schema=[\n",
+    "        bigquery.SchemaField('name', 'STRING'),\n",
+    "        bigquery.SchemaField('post_abbr', 'STRING')\n",
+    "    ],\n",
+    "    skip_leading_rows=1,\n",
+    "    # The source format defaults to CSV, so the line below is optional.\n",
+    "    source_format=bigquery.SourceFormat.CSV\n",
+    ")\n",
+    "uri = 'gs://cloud-samples-data/bigquery/us-states/us-states.csv'\n",
+    "destination_table_ref = dataset.table('us_states_from_gcs')\n",
+    "\n",
+    "# Start the load job\n",
+    "load_job = client.load_table_from_uri(\n",
+    "    uri, destination_table_ref, job_config=job_config)\n",
+    "print('Starting job {}'.format(load_job.job_id))\n",
+    "\n",
+    "load_job.result()  # Waits for table load to complete.\n",
+    "print('Job finished.')\n",
+    "\n",
+    "# Retreive the destination table\n",
+    "destination_table = client.get_table(table_ref)\n",
+    "print('Loaded {} rows.'.format(destination_table.num_rows))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run a parameterized query\n",
```

Comment:
I'd probably want to move this by the other query samples, too.

Reply:
Updated